src/core/Object3D.d.ts

```diff
@@ -135,7 +135,7 @@ export class Object3D extends EventDispatcher {
 	 * Gets rendered into shadow map.
 	 * @default false
 	 */
-	castShadow: boolean;
+	castShadow: undefined | boolean;
```

Comment:
Could this change break existing TS code?

Reply:
I think that loosen a prop from "mandatory" to "optional" will not break in **ts grammer**.
But adding one possible member to a type must break user code in **logic**.

Reply:
Well, this is definitely problematic. In this case, it's probably better to go for `false` as the value for `castShadow`. Since `false` is the default value, any assignment to `castShadow` in `AmbientLight` and `HemisphereLight` can be removed.